### ISSUE TITLE
tagging schema for craft=atelier

### ISSUE THREAD

**Issue by tiptoptom** [openstreetmap/id-tagging-schema#1362]
### OSM Tag(s)

craft=atelier

### How would you like this tag to see supported?

As a Preset

### Label

atelier

### Aliases

art studio

### Link to OSM Wiki page

https://wiki.openstreetmap.org/wiki/Tag:craft%3Datelier

### Status of the Tag

In Use

### Usage of the tag

517

### Regional Tag?

It's mostly used in Europe

**Comment by tordans** [comment]
Thanks for the suggestion.

Does anyone see any reason not to add this?

It does have a relative low usage of ~500+ but looks well defined.
The wiki page is from 2019 so it had time to collect feedback.
And [the wiki page discussions](https://wiki.openstreetmap.org/wiki/Talk:Tag:craft%3Datelier) don't sound like blockers.

**Comment by Kanhaiya-Dixit** [comment]
can you summarize this briefly

**Comment by tiptoptom** [comment]
> can you summarize this briefly. which tag can i use? instead of above tags

I think craft=atelier can be used.

**Comment by tiptoptom** [comment]
What about adding craft=atelier in the [craft field](https://github.com/openstreetmap/id-tagging-schema/blob/main/data/fields/craft.json) as well?

**Comment by srujanaelicherla** [comment]
Hello @tiptoptom @tordans ,

I’ve opened a PR adding support for craft=atelier:
https://github.com/openstreetmap/id-tagging-schema/pull/1700

This adds:

A new preset for ateliers (artist workshops / studios)
An atelier option in the craft field
Search terms: art studio, art workshop, studio

Would appreciate a review when you get a chance 

Thank you!

**Comment by matkoniecz** [comment]
I have no great experience with this kind of POI, but I know that some are really internal/private.

Presumably we would not map atelier in someone's private home (unless it is in some way public/signed/semipublic).

For me https://wiki.openstreetmap.org/wiki/Tag:craft=atelier is really missing it and until it is improved I am not enthusiastic about promoting this tag.

I have now opened https://wiki.openstreetmap.org/wiki/Talk:Tag:craft%3Datelier#Surely_it_would_not_be_for_all_places_where_visual_artists_work%3F
